src/languages/es.ts

```diff
@@ -2624,4 +2624,13 @@ export default {
         taxRateChanged: 'La tasa de impuesto fue modificada',
         taxRequired: 'Falta la tasa de impuesto',
     },
+    videoPlayer: {
+        play: 'Reproducir',
+        pause: 'Pausa',
+        fullscreen: 'Pantalla completa',
+        playbackSpeed: 'Velocidad de reproducción',
```

Comment:
Checked with native speakers from the team, let's slightly adjust these 2:

```suggestion
        pause: 'Pausar',
        fullscreen: 'Pantalla completa',
        playbackSpeed: 'Velocidad',
```